site/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    .addEntry('app', ['./web/assets/scss/librecores.scss',
+        './web/assets/css/bootstrap-social.css',
+        './web/assets/css/font-awesome.min.css',
+        './web/assets/css/librecores.css',
+        './web/assets/scss/bootstrap-librecores.scss'],
+        )
+    //.addEntry('page1', './assets/js/page1.js')
+    //.addEntry('page2', './assets/js/page2.js')
+
+    // will require an extra script tag for runtime.js
+    // but, you probably want this, unless you're building a single-page app
+    .enableSingleRuntimeChunk()
+
+    .cleanupOutputBeforeBuild()
+    .enableSourceMaps(!Encore.isProduction())
+    // enables hashed filenames (e.g. app.abc123.css)
+    .enableVersioning(Encore.isProduction())
```

Comment:
You enable versioned files here (which is good for production), but always include `app.js` above. That probably won't work?

Reply:
@imphil I didn't get what you mean to say here.

Reply:
@aquibbaig You have a fixed link to "build/app.js" which will not work. Use `encore_entry_script_tags`